### ISSUE TITLE
Comfort with physical displays of affection varies by person, not just "culture"

### ISSUE THREAD

**Issue by chaals** [w3c/PWETF#321]
I think we should reframe discussion of physical interactions as questions of respect for individuals rather than ascribing much weight to culture.

In expected behaviours we say (emphasis mine)

> Be aware that displays of affection may complicate professional relationships. For some **cultures**, overtly friendly disposition towards another participant involving body contact (e.g.: hugging, touching on the arm or shoulder, or kissing) is uncommon and may be perceived as an invasion of personal space, or as <a>unwelcome advances</a>.

Where I think it should read "for some people..."

It is not just a "cultural artifact", and while there may be such tendencies in various cultures, in practice it is a question of individual relations and respect for individuals.

There is no reason I should expect that it is automatically OK to kiss a french or spanish person "because we all do that" (speaking as someone who has been living in both of those places and noting that the predominant social conventions are indeed that people kiss as a greeting, and in Spain often touch the person one is talking to a lot).

For that matter, it is not appropriate to exclude physical contact with someone (for example, a person arriving for dinner in Spain as part of a group) **just because** that is not what would happen "in their culture" (say, I arrive for dinner with a group of local colleagues in Tokyo). 

While it is vitally important to be sensitive to people's differing acceptance of physical interactions, especially people one doesn't know, some terms we use to describe "treating people in a different way based on one's perception of their cultural background" may clarify that this issue is complex. (I am not referring to terms such as "situational awareness", "cultural sensitivity", or "respect for individuals", so much as assorted "ism"s and their ilk).

**Comment by dbooth-boston** [comment]
This seems like a good suggestion to me.

However, I'm puzzled that a PR was created for this issue before it was even discussed.  I thought the preferred process was to discuss first in the issues list, and then create a PR if there appeared to be agreement on the issue, because discussion becomes scattered and hard to follow as soon as a PR is created.
